recipes/service_server.rb

```diff
@@ -39,7 +39,7 @@
 
 %w{ obs-source_service }.each do |package_name|
   package package_name do
-    action :install
+    action :upgrade
```

Comment:
You remember correctly, and it's not yet fixed unfortunately: https://github.com/chef/chef/issues/4863
I changed it anyway in the hope that someday it will work...